src/runtime/config/configuration-qemu-sev.toml.in

```diff
@@ -647,4 +647,4 @@ service_offload = @DEFSERVICEOFFLOAD@
 #
 # Keys can be remotely provisioned. The Kata agent fetches them from e.g.
 # a HTTPS URL:
-#provision=https://my-key-broker.foo/tenant/<tenant-id>
\ No newline at end of file
+#provision=https://my-key-broker.foo/tenant/<tenant-id>
```

Comment:
Unrelated, but do we still want these sample options at the bottom of each config? They don't really match CCv0 or watch is planned for main and currently I don't think they do anything.

Reply:
Yep, I didn't even notice this was part of the first PR (sorry), and I dropped changes related to this on this version.

If we're not using this anywhere, we should drop this part for sure.